src/qibocal/protocols/characterization/randomized_benchmarking/standard_rb.py

```diff
@@ -1,71 +1,101 @@
 from copy import deepcopy
 from dataclasses import dataclass
-from typing import Iterable, List, Optional, Tuple
+from typing import Iterable, List, Optional, Tuple, TypedDict, Union
 
 import numpy as np
+import pandas as pd
 import plotly.graph_objects as go
-from qibo.noise import NoiseModel
+import qibo
 
-from qibocal.auto.operation import Routine
+from qibocal.auto.operation import Parameters, Results, Routine
+from qibocal.config import log, raise_error
 from qibocal.protocols.characterization.randomized_benchmarking import noisemodels
-from qibocal.protocols.characterization.randomized_benchmarking.circuit_tools import (
+
+from .circuit_tools import (
     add_inverse_layer,
     add_measurement_layer,
     embed_circuit,
     layer_circuit,
 )
-from qibocal.protocols.characterization.randomized_benchmarking.result import (
-    DecayWithOffsetResult,
-    plot_decay_result,
-)
-from qibocal.protocols.characterization.randomized_benchmarking.utils import (
-    extract_from_data,
-    random_clifford,
-)
-
-from .data import RBData
-from .params import RBParameters
+from .fitting import exp1B_func, fit_exp1B_func
+from .utils import extract_from_data, random_clifford
 
 NPULSES_PER_CLIFFORD = 1.875
 
 
-@dataclass
-class StandardRBResult(DecayWithOffsetResult):
-    """Inherits from `DecayWithOffsetResult`, a result class storing data and parameters
-    of a single decay with statistics.
+class DepthsDict(TypedDict):
+    start: int
+    stop: int
+    step: int
 
-    Adds the method of calculating a fidelity out of the fitting parameters.
-    TODO calculate SPAM errors with A and B
-    TODO calculate the error of the fidelity
 
-    """
+@dataclass
+class StandardRBParameters(Parameters):
+    """Standard Randomized Benchmarking runcard inputs."""
+
+    nqubits: int
+    """The amount of qubits on the chip """
+    qubits: list
+    """A list of indices which qubit(s) should be benchmarked """
+    depths: Union[list, DepthsDict]
+    """A list of depths/sequence lengths. If a dictionary is given the list will be build."""
+    niter: int
+    """Sets how many iterations over the same depth value."""
+    nshots: int
+    """For each sequence how many shots for statistics should be performed."""
+    noise_model: str = ""
+    """For simulation purposes, string has to match what is in
+    :mod:`qibocal.protocols.characterization.randomized_benchmarking.noisemodels`"""
+    noise_params: Optional[list] = None
```

Comment:
```suggestion
    noise_params: Optional[list] = field(default_factory=list)
```
In this way you can drop the lines to initialize `noise_params` in the `__post_init__`.
See comment below.